RecoEcal/EgammaClusterAlgos/src/PFECALSuperClusterAlgo.cc

```diff
@@ -392,6 +392,22 @@ buildSuperCluster(CalibClusterPtr& seed,
 	<< std::endl;
     }    
   }
+
+  //temporary fix for HGCal issue
```

Comment:
the comment is now out of sync. Based on the discussion in this PR, the fix is no longer temporary.